docs/Users/Tutorials/Tutorial.rst

```diff
@@ -232,7 +232,9 @@ any routines yield false positives.
 Ignoring Files
 ~~~~~~~~~~~~~~
 
-*coala* lets you ignore whole files through the ``ignore`` setting:
+*coala* lets you ignore whole files through the ``ignore`` setting. It offers
+recursive globs, so you can use ``**`` to match all directories and
```

Comment:
well the rest of the glob syntax is usually known, this is the most useful "weird" think we offer on top